### PR TITLE
Refactor README.md

### PR DESCRIPTION
**Pull request by mkgrgis**
By https://github.com/ibarwick/firebird_fdw/blob/master/README.md example of advanced FDW documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,157 +1,279 @@
...
+Identifier case handling
+------------------------
+
+PostgreSQL folds identifiers to lower case by default, JDBC data source can have different behaviour.
+Rules and problems **yet not tested and described**.
+
+Generated columns
+-----------------
+
+Behavoiur within generated columns **yet not tested and described**.
```
Comment (weiting1lee) [inline README.md:208]: behavior/behaviour spelling miss

File: README.md [hunk 2]
```diff
@@ -1,157 +1,279 @@
...
+
+PostgreSQL folds identifiers to lower case by default, JDBC data source can have different behaviour.
+Rules and problems **yet not tested and described**.
+
+Generated columns
+-----------------
+
+Behavoiur within generated columns **yet not tested and described**. 
+
+`jdbc_fdw` ptentially can provide support for PostgreSQL's generated
```
Comment (weiting1lee) [inline README.md:210]: potentially spelling miss

Comment (mkgrgis) [reply README.md]: Thanks, @weiting1lee !

**Comment by mkgrgis** [comment]
> @mkgrgis thanks for the pull request. I have reviewed and found some spelling misses, could you please help me fix it?

Yes, @weiting1lee. What about resolving conflict with https://github.com/pgspider/jdbc_fdw/commit/e69cfa37768fcf619891f323afdac6b93fbb9d01#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 ? Who will do it? I can add this information to README.md

**Comment by weiting1lee** [comment]
@mkgrgis
Thanks a lot.
>Who will do it?  I can add this information to README.md
Appreciate if you can, otherwise I'll do it.

**Comment by mkgrgis** [comment]
Added in https://github.com/pgspider/jdbc_fdw/pull/14/commits/e6fa9a058bb97f906d02f374acb012fdadc78480 , @weiting1lee .

**Comment by weiting1lee** [comment]
@mkgrgis
Appreciate for this pull request.
Reviewed and merged.

**Pull request merged**